modules/seo-tools/jetpack-seo.php

```diff
@@ -128,16 +128,20 @@ public function meta_tags() {
 			$obj = get_queried_object();
 
 			$meta['description'] = sprintf(
+				/* translators: first property is an user's display name, the second is the site's title. */
 				_x( 'Read all of the posts by %1$s on %2$s', 'Read all of the posts by Author Name on Blog Title', 'jetpack' ),
-				$obj->display_name,
+				( is_object( $obj ) && property_exists( $obj, 'display_name' ) ) ? $obj->display_name : __( 'the author', 'jetpack' ),
```

Comment:
Is there any reason not to just do this?

```suggestion
				isset( $obj->display_name ) ? $obj->display_name : __( 'the author', 'jetpack' ),
```

`isset()` will suppress any potential warnings about `$obj` not being an object or `->display_name` not existing, and if it happens to exist with null as the value we should probably do "the author" anyway.

Reply:
No, just me blanking re `isset` handling the object immediately.